pkg/tests/tasks/security/authorization/common.go

```diff
@@ -44,15 +44,15 @@ func assertRequestAccepted(t test.TestHelper, ns string, curlCommand string) {
 	})
 }
 
-func assertRequestDenied(t test.TestHelper, ns string, curlCommand string) {
+func assertRequestDenied(t test.TestHelper, ns string, curlCommand string, expectedStatusCode string, expectedStatus string) {
```

Comment:
Unused parameter `expectedStatus`

Reply:
Just remove it.

Reply:
@luksa: removed the unnecessary parameter and pushed the latest change.